apps/sms/test/marionette/lib/views/new-message/view.js

```diff
@@ -77,11 +102,73 @@ NewMessageView.prototype = {
   },
 
   typeMessage: function(message) {
-    this.accessors.messageInput.sendKeys(message);
+    this.composerAccessors.messageInput.sendKeys(message);
+  },
+
+  addAttachment: function() {
+    return this.composerAccessors.addAttachment();
+  },
+
+  typeSubject: function(subject) {
+    this.composerAccessors.subjectInput.sendKeys(subject);
+  },
+
+  showOptions: function() {
+    this.accessors.optionsButton.tap();
+  },
+
+  showSubject: function() {
+    this.showOptions();
+    this.menuAccessors.selectAppMenuOption('Add subject');
+    this.client.helper.waitForElement(this.composerAccessors.subjectInput);
+  },
+
+  hideSubject: function() {
+    this.showOptions();
+    this.menuAccessors.selectAppMenuOption('Remove subject');
+    this.client.helper.waitForElementToDisappear(
+      this.composerAccessors.subjectInput
+    );
+  },
+
+  send: function() {
+    this.composerAccessors.send();
+
+    var ConversationView = require('../conversation/view');
+    var conversationView = new ConversationView(this.client);
+    conversationView.accessors.waitToAppear();
+
+    return conversationView;
+  },
+
+  backToInbox: function() {
+    this.accessors.header.scriptWith(function(header) {
+      var event = document.createEvent('HTMLEvents');
```

Comment:
wondering why you don't use a CustomEvent ? This is what's used in gaia-header too.